python/examples/running-ase-md.py

```diff
@@ -0,0 +1,254 @@
+"""
+Running MD with ASE
+===================
+.. start-body
+"""
+
+# %%
+#
+# .. py:currentmodule:: metatensor.torch.atomistic
+#
+# In this tutorial, we will show how to use a metatensor atomistic model to run
+# Molecular Dynamics (MD) simulation using ASE. ASE is not the only way to run MD with
+# metatensor models, but it is very easy to setup and use.
+#
+# .. TODO: once we have more MD engine, add link to their doc here
+
+from typing import Dict, List, Optional
+
+# tools to run the simulation
+import ase.build
+import ase.md
+import ase.md.velocitydistribution
+import ase.units
+
+# tools for visualization
+import ase.visualize.plot
+import matplotlib.pyplot as plt
+
+# the usuals suspects
+import numpy as np
+import torch
+
+from metatensor.torch import Labels, TensorBlock, TensorMap
+from metatensor.torch.atomistic import (
+    MetatensorAtomisticModel,
+    ModelCapabilities,
+    ModelOutput,
+    System,
+)
+
+# Integration with ASE calculator for metatensor atomistic models
+from metatensor.torch.atomistic.ase_calculator import MetatensorCalculator
+
+
+# %%
+#
+# The energy model
+# ----------------
+#
+# To run simulations, we'll need a model. For simplicity and keeping this tutorial
+# short, we'll use a model for `Einstein's solid`_, where all atoms oscillate around
+# their equilibrium position as harmonic oscillators, but don't interact with one
+# another. The energy of each atoms is given by:
+#
+# .. math::
+#
+#     E_i = k * (\vec{r_i} - \vec{r_i}^0)^2,
+#
+# where :math:`k` is a harmonic force constant, :math:`\vec{r_i}` the position of atom
+# :math:`i` and :math:`\vec{r_i}^0` the position of atom :math:`i` at equilibrium. The
+# energy of the whole system is then
+#
+# .. math::
+#
+#     E = \sum_i E_i.
+#
+# .. _Einstein's solid: https://en.wikipedia.org/wiki/Einstein_solid
+#
+# Let's implement this model! You can check the :ref:`atomistic-tutorial-export`
+# tutorial for more information on how to define your own custom models.
+
+
+class HarmonicModel(torch.nn.Module):
```

Comment:
Maybe a docstring here what k - a positive constant (in units of eV/Å^2) and equilibrium_positions are.